config/types.go

```diff
@@ -116,7 +116,7 @@ type ServiceConfig struct {
 	ShmSize       int64                `yaml:"shm_size,omitempty"`
 	StopSignal    string               `yaml:"stop_signal,omitempty"`
 	VolumeDriver  string               `yaml:"volume_driver,omitempty"`
-	Volumes       []string             `yaml:"volumes,omitempty"`
+	Volumes       *yaml.Volumes        `yaml:"volumes,omitempty"`
```

Comment:
hum good question, but I don't think so, it will be converted to v2 anyway and it's this struct that is used everywhere else so, I don't see why we would want to update v1 structs